src/pal/pal_open_enclave.h

```diff
@@ -66,6 +67,17 @@ namespace snmalloc
     {
       oe_memset_s(p, size, 0, size);
     }
+
+    /**
+     * Source of Entropy
+     */
+    static uint64_t get_entropy64()
+    {
+      uint64_t result = 0;
+      if (oe_random(&result, sizeof(result)) != 0)

```

Comment:
```suggestion
      if (oe_random(&result, sizeof(result)) != OE_OK)
```

Reply:
Do we have OE_OK in scope when we build snmalloc for OE?  I'll add it to the simulatation layer, but if you could check it works when you build that would be great.

Reply:
If we have oe_random, then we have oe_result_t. If we're missing one or the other, I'll add them.